docs/releases/2.1.txt

```diff
@@ -155,7 +155,8 @@ Internationalization
 Management Commands
 ~~~~~~~~~~~~~~~~~~~
 
-* ...
+* The new :option:`inspectdb --include-views` option allows creating models
```

Comment:
Not sure why you get this change into the PR. Perhaps something about the tracking branch? Anyway, it shouldn't be here.

Reply:
This looks like an error in conflict resolution?